data/hails.txt

```diff
@@ -1002,12 +1002,197 @@ phrase "friendly civilian"
 		"."
 		"!"
 
+phrase "friendly civilian"
+	word
+		"Once a militia officer "
+		"Some crazy person down in the south "
+		"Some crazy militia officer "
+		"Walking into the spaceport, someone "
+	word
+		"asked me if I would help her fight local pirates. That's too dangerous for me!"
+		"pointed a gun at the back of my head. I almost had to change my pants!"
+		"almost broke my arm when I tried to karate chop her. I left that place in a hurry!"
+
+phrase "friendly civilian"
+	word
+		"Two "
+	word
+		"merchants"
+		"pirates"
+		"aliens"
+		"Quarg"
+		"Navy officers"
+		"Syndicate executives"
+		"militia officers"
+	word
+		" walk into a spaceport"
+		" land on a planet"
+		" hyperdrive into a system"
+		" get disabled in space"
+		""
+	word
+		". . . "
```

Comment:
I'm not sure about this part.
Are there really supposed to be spaces separating the ellipsis?